HW_1/HW1.py

```diff
@@ -0,0 +1,141 @@
+# Complementator
+transcribe_dict = dict(A='A', T='U', G='G', C='C', a='a', t='u', g='g', c='c')
+complementDNA_dict = dict(A='T', T='A', G='C', C='G', a='t', t='a', g='c', c='g')
+complementRNA_dict = dict(A='U', U='A', G='C', C='G', a='u', u='a', g='c', c='g')
```

Comment:
Обычно такие константы, которые мы используем во всей программе, принято называть заглавными буквами
```suggestion
TRANSCRIBE_DICT = dict(A='A', T='U', G='G', C='C', a='a', t='u', g='g', c='c')
DNA_COMPL_DICT = dict(A='T', T='A', G='C', C='G', a='t', t='a', g='c', c='g')
RNA_COMPL_DICT = dict(A='U', U='A', G='C', C='G', a='u', u='a', g='c', c='g')
```